counter-sink/src/main/resources/application.yml

```diff
@@ -5,4 +5,4 @@ spring:
   cloud:
     stream:
       bindings:
-        input: counter
+        input: ${remote.name:application:${server.port}}
```

Comment:
I don't understand this (I'm not even sure how it is parsed, with those two colons).
What is `server.port` doing here? is it a way to provide unicity? I thought we agreed `server.port` would be 0 in this file, to be overridden by s-c-data. Is this the intent here, _i.e._ when `remote.name` is not provided, end up being `application:1234` ?